commit 92d4dca49dc72e57f7cb58fba81801eae884ebcf
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Sat Feb 12 01:49:52 2022 +0100

    fix version to be apk-compatible
    
    this is needed for correct pkg-config scanning

diff --git a/meson.build b/meson.build
index b13b1cf..d7693d3 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project(
   'totem', 'c',
-  version: '41.alpha.0',
+  version: '41.0.0',
   license: 'GPL2+ with exception',
   default_options: 'buildtype=debugoptimized',
   meson_version: '>= 0.50.0'
@@ -235,9 +235,7 @@ endif
 
 meson.add_install_script('meson_post_install.py')
 
-is_stable = (totem_minor_version != 'alpha' and
-  totem_minor_version != 'beta' and
-  totem_minor_version != 'rc')
+is_stable = false
 if is_stable
   meson.add_dist_script(
     find_program('check-news.sh').path(),
